discovery/consul/consul.go

```diff
@@ -20,6 +21,8 @@ func init() {
 type Consul struct {
```

Comment:
Agreed on this but I'm going to cover that under https://github.com/joyent/containerpilot/issues/317.